docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/evaluator-configuration-file.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Evaluator configuration file
+description: Reference information for evaluator configuration file.
+template: howto-guide-template
+---
+
+The configuration file allows to configure evaluator tool for specific project.
+
+## Problem description
+
+The built-in evaluator configuration is not enough for some projects. For example some errors are not fixable and should be ignored. 
+Need to provide the way to configure evaluator checkers on project level.
+
+## Evaluator configuration
+
+To configure the evaluator need to place `tooling.yml` in the project root directory.
+
+```yaml
+# tooling.yml
+
+evaluator:
+  # evaluator options
+  ...
+
+```
+
+## Ignoring checkers errors
+
+To ignore errors need to specify regular expressions in `ignoreErrors` section in configuration file. Each error message that matches these regular expressions will be ignored by evaluator.
+You can specify rules both for some specific checkers or globally for all of them.
```

Comment:
Previously, you've mentioned that in this config, we can add *errors* to ignore.
Here, you say that we can add *checkers to ignore.

So what is it exactly that we can ignore?